opt/scopeserver/dataserver/modules/gserver/GServer.py

```diff
@@ -580,19 +580,57 @@ def getMarkerGenes(self, request, context):
             md_cmm = md_clustering["clusterMarkerMetrics"]
 
             def create_cluster_marker_metric(metric):
-                cluster_marker_metrics = loom.get_cluster_marker_metrics(
-                    clustering_id=request.clusteringID, cluster_id=request.clusterID, metric_accessor=metric["accessor"]
+                return loom.get_cluster_marker_metrics(
+                    clustering_id=request.clusteringID,
+                    cluster_id=request.clusterID,
+                    metric_accessor=metric["accessor"]
                 )
+
+            def to_pb(metric):
```

Comment:
Can you think of a clearer name than `to_pb`?

Reply:
Following @KrisDavie style, I could use `protoize_cluster_marker_metric` ; Does that sound good ?

Reply:
Yes I prefer that